test/conftest.py

```diff
@@ -56,6 +57,7 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("modules", [{p: all_modules[p] for p in module_names}])
         metafunc.parametrize("terse", [metafunc.config.getoption("--terse")])
         metafunc.parametrize("silent", [metafunc.config.getoption("--silent")])
+        metafunc.parametrize("traceback", [metafunc.config.getoption("--traceback")])
```

Comment:
This actually needs to go under a new block, `if metafunc.function.__name__ == "test_regressions"` and I think it'll be `opt_traceback`: https://github.com/cclib/cclib/blob/67be86feb8d97eb123c268b86323afec008d6dc5/test/regression.py#L3294
and you'll probably get the cryptic "default argument" error I was getting.